### ISSUE TITLE
golbin and goniomish quest bug 

### ISSUE THREAD

**Issue by mayras321** [dalaranwow/dalaran-wow#5627]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. choosing goblin engineering. took the quest and it just disappeared and not in my quest log and I am not a goblin engineer 
2. tried same thing for gonomeish quest and I am not a gonomish engineer either and it doesn't show in my book 
3. I hav a specific bomb goblin that says I require goblin engineering same for the gonomish 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Kookaburra129** [comment]
+1

Made multiple tickets regarding this and also have had email chains requesting assistance. Other than the original questline, the item used to switch between goblin & gnomish engineering is still not working.

Tickets & emails get answered saying a fix is coming in a few days - but nothing seems to be happening.

**Comment by Lothloryen** [comment]
Hello there,

This will be fine after the next maintenance.

**Comment by mayras321** [comment]
Thank you I will get back in an see if it’s fixed that was at least 2 years ago from when I started asking questions abought that

**Comment by mayras321** [comment]
Glad to here u have a fix for it
